deployment/security-server/docker/compose.yaml

```diff
@@ -0,0 +1,217 @@
+      - xroad-network
+    depends_on:
+      db-serverconf-init:
+        condition: service_completed_successfully
+      configuration-client:
+        condition: service_healthy
+      proxy:
+        condition: service_healthy
+  signer:
+    image: ${SIGNER_IMG}
+    pull_policy: always
+    environment:
+      - XROAD_COMMON_RPC_CHANNEL_CONFIGURATION_CLIENT_HOST=configuration-client
+    healthcheck:
+      interval: 5s
+      retries: 40
+      test: [ "CMD", "curl", "-f", "-k", "http://localhost:4099/q/health" ]
+    networks:
+      - xrd-ss-network
+      - xroad-network
+    depends_on:
+      db-serverconf-init:
+        condition: service_completed_successfully
+      configuration-client:
+        condition: service_healthy
+    deploy:
+      resources:
+        reservations:
+          memory: 64M
+        limits:
+          memory: 320M
+  proxy:
+    image: ${PROXY_IMG}
+    pull_policy: always
+    environment:
+      - XROAD_COMMON_RPC_CHANNEL_CONFIGURATION_CLIENT_HOST=configuration-client
+      - XROAD_COMMON_RPC_CHANNEL_SIGNER_HOST=signer
+      - XROAD_DB_MESSAGELOG_HIBERNATE_CONNECTION_URL=jdbc:postgresql://db-messagelog:5432/messagelog
+      - XROAD_DB_MESSAGELOG_HIBERNATE_CONNECTION_DRIVER_CLASS=org.postgresql.Driver
+      - XROAD_DB_MESSAGELOG_HIBERNATE_CONNECTION_USERNAME=messagelog
+      - XROAD_DB_MESSAGELOG_HIBERNATE_CONNECTION_PASSWORD=secret
+      - XROAD_DB_SERVERCONF_HIBERNATE_CONNECTION_URL=jdbc:postgresql://db-serverconf:5432/serverconf
+      - XROAD_DB_SERVERCONF_HIBERNATE_CONNECTION_DRIVER_CLASS=org.postgresql.Driver
+      - XROAD_DB_SERVERCONF_HIBERNATE_CONNECTION_USERNAME=serverconf
+      - XROAD_DB_SERVERCONF_HIBERNATE_CONNECTION_PASSWORD=secret
+    healthcheck:
+      interval: 5s
+      retries: 40
+      test: [ "CMD", "curl", "-f", "-k", "http://localhost:4099/q/health" ] #TODO verify that endpoint returns 200
+    networks:
+      - xrd-ss-network
+      - xroad-network
+    depends_on:
+      db-serverconf-init:
+        condition: service_completed_successfully
+      signer:
+        condition: service_healthy
+    ports:
+      - "0:9999" # debug port
+    deploy:
+      resources:
+        reservations:
+          memory: 64M
+        limits:
+          memory: 512M
+  ui:
+    image: ${PROXY_UI_IMG}
+    pull_policy: always
+    environment:
+      - XROAD_COMMON_RPC_CHANNEL_CONFIGURATION_CLIENT_HOST=configuration-client
+      - XROAD_COMMON_RPC_CHANNEL_SIGNER_HOST=signer
+      - XROAD_COMMON_RPC_CHANNEL_PROXY_HOST=proxy
+      - XROAD_COMMON_RPC_CHANNEL_ENV_MONITOR_HOST=monitor
+      - XROAD_DB_SERVERCONF_HIBERNATE_CONNECTION_URL=jdbc:postgresql://db-serverconf:5432/serverconf
+      - XROAD_DB_SERVERCONF_HIBERNATE_CONNECTION_DRIVER_CLASS=org.postgresql.Driver
+      - XROAD_DB_SERVERCONF_HIBERNATE_CONNECTION_USERNAME=serverconf
+      - XROAD_DB_SERVERCONF_HIBERNATE_CONNECTION_PASSWORD=secret
+      - XROAD_PROXY_UI_API_SECURITY_SERVER_URL=https://proxy:8443
+#    volumes:
```

Comment:
I'd say these compose files are now specialized for tests, meaning that it should be cleaned from such comments (multiple places).